README.adoc

```diff
@@ -0,0 +1,478 @@
+= Spring Security Docs Home
+ifndef::env-github[:toc:]
+ifdef::env-github[]
+:important-caption: :exclamation:
+:note-caption: :paperclip:
+endif::[]
+
+This README describes the processes and tools used to build the documentation for production as well as how to generate a local preview of the documentation.
+
+== Overview
+
+The Spring Security reference docs are generated using Antora.
+The Gradle Antora Plugin is used as the primary interface to Antora.
+
+You're viewing the playbook branch for the Spring Security project.
+The playbook branch hosts the docs build used to build and publish the production docs site, and is thus the documentation home base.
+If you're a docs developer, you'll mostly work in this branch.
+If you're a writer, you'll mostly work in software branches.
+
+=== Layout
+
+The playbook branch, named *docs-build*, hosts the primary documentation build.
+The documentation itself is located in a dedicated subproject in each software branch (i.e., the docs are stored alongside the code).
+Software branches, also referred to as release line branches or content branches, follow the pattern `major.minor.x` (e.g., 6.0.x), or `main` for the latest release line.
+Software tags follow the pattern `major.minor.patch` (e.g., 6.0.1).
+The latest version of the docs for each release line is always sourced from a tag.
+The release line branches host the prerelease materials for the next version.
+
+=== Builds
+
+The build for the production site is stored at the root of the playbook branch.
+This branch also holds the search crawler configuration and runner for the production site.
+This build is only needed for building the production site (or for developing and testing the docs build itself).
+The build for each content branch is located in a subproject, typically the folder named _docs_.
+This is the build that authors use to preview the docs for a single version when writing content.
+
+Regardless of how the docs site is built, Antora is configured to run a separate command to compute the version of the docs per git reference.
+This command is run by an Antora extension named Antora Collector.
+Using Collector allows the version of the docs to be maintained centrally in the _gradle.properties_ file in each git reference.
```

Comment:
What about maven users? Where is the version maintained? I guess it is one of the properties of the docs's `pom.xml`, is that correct? We need to update this in the readme.